presto-main/src/main/java/com/facebook/presto/cost/StatsAndCosts.java

```diff
@@ -77,15 +79,16 @@ public StatsAndCosts getForSubplan(PlanNode root)
         return new StatsAndCosts(filteredStats.build(), filteredCosts.build());
     }
 
-    public static StatsAndCosts create(PlanNode root, StatsProvider statsProvider, CostProvider costProvider)
+    public static StatsAndCosts create(PlanNode root, StatsProvider statsProvider, CostProvider costProvider, Session session, boolean useCache)
```

Comment:
maybe add a comment that useCache should only be false for tests

Reply:
Yeah, the argument was added for plan tests only. Since we decided to add a session property to control the behaviour, I decided to remove this argument here, as it will be turned off though session property for the tests now.